__tests__/app.test.js

```diff
@@ -190,4 +190,82 @@ describe('Testing app', () => {
           .expect(204);
         });
     });
+    describe('POST', () => {
+        test('status:201, responds with a new comment added to an article', () => {
+          const testComment = {
+            article_id: 1,

```

Comment:
The request body should only accept two things looking at the ticket:
`Request body accepts:

an object with the following properties:
username
body`

The article Id should be handled from the params.